src/services/item/plugins/published/service.ts

```diff
@@ -127,6 +158,10 @@ export class ItemPublishedService {
 
     const item = await this.itemService.get(actor, repositories, itemId, PermissionLevel.Admin);
 
+    if (itemTypeNotAllowedToBePublished(item.type)) {
```

Comment:
I wonder whether it would be clearer to export an array from sdk and check if the type is included here. It's a bit "black boxed" to import a function.

Reply:
I think it is preferable to use a function to voluntary hide the implementation to the service. In this way, the service delegate the check to another class (it is now a class, as @spaenleh suggested) and only ask if the given type is valid. It also allow us to reuse it in the frontend too. Finally, I don't think that its a problem to hide the valid types in the backend, if we really need to know, we can check in the SDK.

What do you think @pyphilia, @spaenleh, @Defferrard ? I'm open to argumentation ;)

Reply:
I am ok with this black box.